.github/CONTRIBUTING.md

```diff
@@ -34,7 +34,15 @@ If the bug report is missing this information then it'll take us longer to fix t
 
 ## Submitting a Pull Request
 
-Submitting a pull request is fairly simple, just make sure it focuses on a single aspect and doesn't manage to have scope creep and it's probably good to go. It would be incredibly lovely if the style is consistent to that found in the project. This project follows PEP-8 guidelines (mostly) with a column limit of 125.
+Submitting a pull request is fairly simple, just make sure it focuses on a single aspect and doesn't manage to have scope creep and it's probably good to go. It would be incredibly lovely if the style is consistent to that found in the project. This project follows PEP-8 guidelines (mostly) with a column limit of 100.
+
```

Comment:
Not entirely sure on the scope of this pull request so either of the two depending which suits most.

It should however be noted that for ongoing backward compatibility issues we will not accept pull requests which utilize an auto-formatter to change the library at this time.

Or:
The library is auto formatted using ``black`` with a configuration of <Insert specific config here>